wasm/src/bindings.cpp

```diff
@@ -48,440 +51,446 @@ extern rive::Factory* jsFactory();
-  std::vector<unsigned char> rv;
+rive::File* load(emscripten::val byteArray)
+{
+    std::vector<unsigned char> rv;
 
-  const auto l = byteArray["byteLength"].as<unsigned>();
-  rv.resize(l);
+    const auto l = byteArray["byteLength"].as<unsigned>();
+    rv.resize(l);
 
-  emscripten::val memoryView{emscripten::typed_memory_view(l, rv.data())};
-  memoryView.call<void>("set", byteArray);
-  return rive::File::import(rv, jsFactory()).release();
+    emscripten::val memoryView{emscripten::typed_memory_view(l, rv.data())};
+    memoryView.call<void>("set", byteArray);
+    return rive::File::import(rv, jsFactory()).release();
 }
 
-rive::Mat2D computeAlignment(rive::Fit fit, rive::Alignment alignment, rive::AABB orig, rive::AABB dest) {
-  return rive::computeAlignment(fit, alignment, orig, dest);
+rive::Alignment convertAlignment(JsAlignment alignment)
+{
+    switch (alignment)
+    {
+        case JsAlignment::topLeft:
+            return rive::Alignment::topLeft;
+        case JsAlignment::topCenter:
+            return rive::Alignment::topCenter;
+        case JsAlignment::topRight:
+            return rive::Alignment::topRight;
+        case JsAlignment::centerLeft:
+            return rive::Alignment::centerLeft;
+        case JsAlignment::center:
+            return rive::Alignment::center;
+        case JsAlignment::centerRight:
+            return rive::Alignment::centerRight;
+        case JsAlignment::bottomLeft:
+            return rive::Alignment::bottomLeft;
+        case JsAlignment::bottomCenter:
+            return rive::Alignment::bottomCenter;
+        case JsAlignment::bottomRight:
+            return rive::Alignment::bottomRight;
+    }
+    return rive::Alignment::center;
 }
 
-rive::Vec2D mapXY(rive::Mat2D invertedMatrix, rive::Vec2D canvasVector) {
-  return invertedMatrix * canvasVector;
+rive::Mat2D computeAlignment(rive::Fit fit, JsAlignment alignment, rive::AABB orig, rive::AABB dest)
+{
+    return rive::computeAlignment(fit, convertAlignment(alignment), orig, dest);
 }
 
-class DynamicRectanizer {
-public:
-  DynamicRectanizer(int maxAtlasSize) :
-      m_Rectanizer(SkISize(),
-                   maxAtlasSize,
-                   GrDynamicRectanizer::RectanizerAlgorithm::kSkyline) {}
-
-  void reset(int initialWidth, int initialHeight) {
-      m_Rectanizer.reset({initialWidth, initialHeight});
-  }
-
-  int addRect(int width, int height) {
-      SkIPoint16 loc;
-      if (!m_Rectanizer.addRect(width, height, &loc)) {
-          return -1;
-      }
-      return (loc.y() << 16) | loc.x();
-  }
-
-  int drawWidth() const { return m_Rectanizer.drawBounds().width(); }
+rive::Vec2D mapXY(rive::Mat2D invertedMatrix, rive::Vec2D canvasVector)
+{
+    return invertedMatrix * canvasVector;
+}
 
-  int drawHeight() const { return m_Rectanizer.drawBounds().height(); }
+class DynamicRectanizer
+{
+public:
+    DynamicRectanizer(int maxAtlasSize) :
+        m_Rectanizer(SkISize::Make(1, 1),
```

Comment:
This was causing an assert (it would initialize with 0,0 prior to this change) @csmartdalton